### PR TITLE
Monk Changes

### PR DESCRIPTION
**Pull request by JhinHandaGay21**
## About The Pull Request
https://github.com/Rotwood-Vale/Ratwood-Keep/issues/776

## Why It's Good For The Game
The monk will no longer be so crappy for other races besides dwarves and Orcs

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/monk.dm [hunk 1]
```diff
@@ -41,4 +41,6 @@
 		H.change_stat("constitution", 1)
 		H.change_stat("speed", 2)
 		H.change_stat("perception", -1)
-		ADD_TRAIT(H, TRAIT_DODGEEXPERT, TRAIT_GENERIC)
```
[the remark this answers is not in the document]
Comment (JhinHandaGay21) [reply code/modules/jobs/job_types/roguetown/adventurer/types/combat/monk.dm:44]: The fast guys will be able to dodge somehow, but the slow ones rely on their constitution and strength

**Comment by ikrarkjr** [comment]
Least obvious powercreep

**Comment by wow50000** [comment]
I actually like this change

**Comment by Useroth** [comment]
That just makes him an unarmed berserker

**Comment by JhinHandaGay21** [comment]
> That just makes him an unarmed barbarian

Then what can you suggest?

**Pull request closed without merging**